digiroad2-oracle/src/main/resources/db/migration/V236__create_LengthOfRoadAxis_properties.sql

```diff
@@ -0,0 +1,205 @@
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Loppu päivämäärä';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Loppu päivämäärä', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Alku päivämäärä';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Alku päivämäärä', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Jyrsitty';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Jyrsitty', 'db_migration_v236', SYSDATE);
+    END IF;
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Kuntanumero';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Kuntanumero', 'db_migration_v236', SYSDATE);
+    END IF;
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Kunto';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Kunto', 'db_migration_v236', SYSDATE);
+    END IF;
+END;
+
+-- add propeties
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'single_choice', 1, 'db_migration_v236', 'regulatory_number',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Asetusnumero'));
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'number', 0, 'db_migration_v236', 'lane_number',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Kaistannumero'));
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'string', 0, 'db_migration_v236', 'lane_type',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Kaistantyyppi'));
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'number', 1, 'db_migration_v236', 'lane_location',
+        (select max(id) from LOCALIZED_STRING where VALUE_FI = 'Suhteellinen sijainti'));
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'string', 0, 'db_migration_v236', 'municipal_id',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Kuntanumero'));
```

Comment:
You don't need this as property because we already have one field at the Asset table for this who can be used (Municipality Code).